be/src/storage/rowset_update_state.h

```diff
@@ -46,6 +47,16 @@ class RowsetUpdateState {
     size_t _memory_usage = 0;
     int64_t _tablet_id = 0;
 
+    // states for partial update
+    EditVersion _read_version;
+    uint32_t _next_rowset_id = 0;
+    struct PartialUpdateState {
+        vector<uint64_t> src_rss_rowids;
```

Comment:
What's the difference between rssids and rss_rowids?

Reply:
rssid: rowset segment id  uint32
rss_rowid: <rssid uint32, rowid uint32> uint64